yarn.lock

```diff
@@ -2000,45 +2000,53 @@
+  integrity sha512-ZCXzXNHWQ7H0qk/kC+rWzjMWjLzZGzQcDbdpIuy8xJdp4rTpmMkLUseFPby8vhkmIaqxWPwPB6mx/vM7L6JENg==
   dependencies:
     "@babel/runtime" "^7.3.1"
-    "@loaders.gl/loader-utils" "3.0.6"
-    "@loaders.gl/schema" "3.0.6"
-    "@loaders.gl/worker-utils" "3.0.6"
+    "@loaders.gl/loader-utils" "3.0.8"
+    "@loaders.gl/schema" "3.0.8"
+    "@loaders.gl/worker-utils" "3.0.8"
     draco3d "1.4.1"
 
-"@loaders.gl/gis@3.0.6", "@loaders.gl/gis@^3.0.6":
-  version "3.0.6"
-  resolved "https://registry.yarnpkg.com/@loaders.gl/gis/-/gis-3.0.6.tgz#000184995ce9377186e723235546b8158b4ef19d"
-  integrity sha512-WF4OVZ3BBJLP6n0cqKgdRwB5PLBJHqQIFesiVbimC0IhMe17IhtYIvEi0Hw9kY7YHCVFtZ0B/U13P5JRJWZDsA==
+"@loaders.gl/gis@3.0.8", "@loaders.gl/gis@^3.0.8":
+  version "3.0.8"
+  resolved "https://registry.yarnpkg.com/@loaders.gl/gis/-/gis-3.0.8.tgz#8f4f74914a4def74e9ef87eac6ddbe1224220e49"
+  integrity sha512-7NL+lIb7NezlMupYskVil6M3RZunXJl+TyaVAW82GLbzPSOq+m/G7h3+z0GBa8iv/U/I+cB5BhSN+GZmvFwqEA==
   dependencies:
-    "@loaders.gl/loader-utils" "3.0.6"
-    "@loaders.gl/schema" "3.0.6"
+    "@loaders.gl/loader-utils" "3.0.8"
+    "@loaders.gl/schema" "3.0.8"
     "@mapbox/vector-tile" "^1.3.1"
     pbf "^3.2.1"
 
-"@loaders.gl/gltf@3.0.6", "@loaders.gl/gltf@^3.0.6":
-  version "3.0.6"
-  resolved "https://registry.yarnpkg.com/@loaders.gl/gltf/-/gltf-3.0.6.tgz#880594acda73571b90048266c3c80553d6b0c78e"
-  integrity sha512-dHy070MYrxC8+I1gnOs9a8KAazvmb9uvl/+VM9Ec8RD3QPZZZvhy2OE3xWDPHJBQt4xBhS4oZXGOgsIu051mVA==
+"@loaders.gl/gltf@3.0.8", "@loaders.gl/gltf@^3.0.8":
+  version "3.0.8"
+  resolved "https://registry.yarnpkg.com/@loaders.gl/gltf/-/gltf-3.0.8.tgz#5959ff30ad2043d755ce8dc958def0eb78044e5d"
+  integrity sha512-4PXWTlqyvlbZE2Vp4iQ+Y87ZO1WuRvSlbImDhygd0hoINfmJ9ObxrFS3yJcpJTu007nWxXorNVEOKyuoo+4Iyw==
   dependencies:
-    "@loaders.gl/core" "3.0.6"
-    "@loaders.gl/draco" "3.0.6"
-    "@loaders.gl/images" "3.0.6"
-    "@loaders.gl/loader-utils" "3.0.6"
+    "@loaders.gl/core" "3.0.8"
+    "@loaders.gl/draco" "3.0.8"
+    "@loaders.gl/images" "3.0.8"
+    "@loaders.gl/loader-utils" "3.0.8"
 
-"@loaders.gl/images@3.0.6", "@loaders.gl/images@^3.0.6":
-  version "3.0.6"
-  resolved "https://registry.yarnpkg.com/@loaders.gl/images/-/images-3.0.6.tgz#038fad37e0ee0c8319cbd440481f92576891c6d0"
-  integrity sha512-vRNayX5X4VNG+hVCE7u2nMwqpe0uu0YVOfxFduOs2XfS3Cqp7v9R7OBtYQbJ2B2yhVSk6Hz1LW1mzS4jJrmyrA==
+"@loaders.gl/images@3.0.8", "@loaders.gl/images@^3.0.8":
+  version "3.0.8"
+  resolved "https://registry.yarnpkg.com/@loaders.gl/images/-/images-3.0.8.tgz#8a05120a2f0097923fce7cc6e4cb0509a1a1d79e"
+  integrity sha512-rO2cIYJYlMs/uO9YSoF4/BEA4p/9xQ3gHZ1sIJkPYVnDqzpbu8nvUjWTQqIdL/MkQBTW8tz3twCdM+B6G9Fa2w==
   dependencies:
-    "@loaders.gl/loader-utils" "3.0.6"
+    "@loaders.gl/loader-utils" "3.0.8"
 
-"@loaders.gl/loader-utils@3.0.6", "@loaders.gl/loader-utils@^3.0.6":
+"@loaders.gl/loader-utils@3.0.6":
```

Comment:
There is still 3.0.6 installed. Run `yarn list "@loaders.gl/*"` to check if the dependency tree is clean. You can also run  `yarn upgrade-interactive "@loaders.gl/*"` to force bump all.

Reply:
Fixed. I had tried to use a tool to do the upgrade and it missed a load of the dependencies :/ Do you have some trick for bumping the loaders/math.gl dependencies across all the packages?